### PR TITLE
mineotaur: enable deployment (IDR-0.3.6)

### PR DESCRIPTION
**Pull request by joshmoore**
* rsync from the production
 * initial work for rsync'ing from `omero_omero` host
 * use the hard-coded sysgro subdir

### CODE REVIEW

File: ansible/idr-mineotaur.yml [hunk 1]
```diff
@@ -1,28 +1,45 @@
+# Load hostvars
+- hosts: "{{ idr_environment | default('idr') }}-omero-hosts"
```
Comment (manics) [inline ansible/idr-mineotaur.yml]: If this was for `omero_rsync_host_ansible` it's no longer needed

File: ansible/idr-mineotaur.yml [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
+
 # Setup IDR Mineotaur server
 - hosts: "{{ idr_environment | default('idr') }}-a-dockermanager-hosts"
 
   tasks:
 
+  - name: Get rsync IP
+    set_fact:
+      omero_rsync_host_ansible : >-
```
Comment (joshmoore) [inline ansible/idr-mineotaur.yml]: My hope is that you would have a suggestion how we want to let `-a-dockermanger`s download from somewhere else. One option, e.g., would be to have `-a-omero`s _also_ serve rsync.

File: ansible/idr-mineotaur.yml [hunk 3]
```diff
@@ -1,28 +1,45 @@
...
       name: mineotaur
       image: imagedata/mineotaur
       entrypoint: java
       volumes:
-        - /tmp/mineotaur:/mineotaur
+        - /tmp/mineotaur/idr0001gramlsysgroscreenA:/mineotaur
       ports:
         - "8080:8080"
-      command: ["-Xmx8g", "-Djava.security.egd=file:/dev/./urandom", "-jar", "/home/build/src/target/Mineotaur-1.0.1.jar", "-start", "/mineotaur"]
+      command: "-Xmx8g -Djava.security.egd=file:/dev/./urandom -jar /home/build/src/target/Mineotaur-1.0.1.jar -start /mineotaur"
```
Comment (joshmoore) [inline ansible/idr-mineotaur.yml]: Hmmm.... I'm not sure. I assume nothing.

**Comment by manics** [comment]
I'll attempt to test this with #16

**Comment by joshmoore** [comment]
Merging for IDR-0.3.6 (as state of prod36)

**Pull request merged**